server/odc-migrate/src/main/resources/migrate/common/V_3_3_0_1__add_flow.sql

```diff
@@ -31,7 +31,8 @@ create table if not exists `flow_instance_node_approval` (
     `is_start_endpoint` tinyint NOT NULL COMMENT 'Describes whether this endpoint is the starting endpoint of the process',
     `is_end_endpoint` tinyint NOT NULL COMMENT 'Describes whether this endpoint is the end endpoint of the process',
     `flow_instance_id` bigint(20) NOT NULL comment 'Process instance id, references flow_instance(id)',
-    constraint pk_flow_instance_node_approval primary key (`id`)
+    constraint pk_flow_instance_node_approval primary key (`id`),
```

Comment:
it will not works, please do not modify versional script, all scripts start with`V_` will be only executed once

Reply:
changed